src/table.js

```diff
@@ -538,15 +543,105 @@ export function getTypeValidator(colType) {
   }
 }
 
+export function coerceToType(value, type) {
+  const stringValue = typeof value === "string" ? value.trim() : value;
+  switch (type) {
+    case "string":
+      return typeof value === "string"
+        ? stringValue
+        : value || value === 0
+        ? value.toString()
+        : null;
```

Comment:
If I’m reading this correctly, this turns the boolean input value `false` into `null`; I expect the string `"false"` instead. It’s also surprising to me that we trim when coercing a string to a string; trimming seems appropriate when coercing to some _other_ type, e.g. to test for “true” and “false”, but I expect that if the input is already in the desired type the value should not be affected. Lastly I expect that we preserve undefined rather than coercing it to null.

In other words, what if we did this?

```suggestion
      return typeof value === "string" || value == null
        ? value
        : String(value);
```

Also, we should use `String(value)` instead of `value.toString()`; the latter can be overridden to return something that’s not a string, and we want to guarantee that the coerced value is a string (or null or undefined).